tests/image_test.py

```diff
@@ -1227,16 +1227,15 @@ def test_load_img(self, tmpdir):
 
 
     def test_list_pictures(self, tmpdir):
-        vocab = list(string.ascii_lowercase + string.ascii_uppercase +
-                     string.digits + '-')
-        images = []
-
-        for i in range(100):
-            images.append(os.path.join(tmpdir,
-                                       f'{"".join(np.random.choice(vocab,
-                                                                   10))}.png'))
-            image.array_to_img(np.array(255 * np.random.rand(100, 100, 3),
-                               dtype=np.uint8), scale=False).save(images[-1])
+        images.append(os.path.join(tmpdir, 'test.png'))
+        images.append(os.path.join(tmpdir, 'test0.jpg'))
+        images.append(os.path.join(tmpdir, 'test-1.jpeg'))
+        images.append(os.path.join(tmpdir, '2test.bmp'))
+        images.append(os.path.join(tmpdir, '2-test.ppm'))
+        images.append(os.path.join(tmpdir, '3.png'))
```

Comment:
`images = [os.path.join(tmpdir, filename) for filename in filenames]`

Reply:
why 'tif' and 'tiff' are missing here and in the function `list_pictures` ? Is this a mistake or am I missing something?

Reply:
it has been this way since the beginning https://github.com/keras-team/keras-preprocessing/blame/014457b52def2af30bff88809437f8941451b76f/keras_preprocessing/image.py#L542
and I'm not sure we should change it since there is no issue related as we can break some systems with the default behavior. What do you think?